src/sim-host/protocol/socket.js

```diff
@@ -49,6 +49,17 @@ module.exports.initialize = function (pluginHandlers, services) {
         document.location.reload(true);
     });
 
+    socket.on('retheme', function () {
+        var links = document.head.getElementsByTagName('link');
+        var themeLinks = Array.prototype.filter.call(links, function (elt){
+            return elt.href.indexOf("sim-host-theme.css") >= 0;
+        });
```

Comment:
You should just be able to do `document.head.querySelectorAll('link[href=sim-host-theme.css]')` and avoid this loop.